source/main.cpp

```diff
@@ -129,7 +129,7 @@ int main(int argc, char **argv) {
 	int patchfile = 0;
 	int header;
 	rawDataOffset=0;
-	char datName[]="YS:/" DATNAME;
+	char datName[MAX_DATNAME_LEN]="YS:/" DATNAME;
```

Comment:
@Syphurith I did not append \0 to the string, but commits at source/main.cpp:132 would cause a compiler error if path string's length is greater than MAX_DATNAME_LEN-1